.circleci/config.yml

```diff
@@ -54,6 +74,45 @@ jobs:
 
 workflows:
   version: 2
+  weekly_rebase:
+    jobs:
+      - lint:
+          filters:
+            tags:
+              only: /^(v\d*\.\d*\.\d*)(?:-(\w*\d*))?/
+            branches:
```

Comment:
keep commit triggers for `lint` and `test`

Reply:
`test_and_deploy` continues to trigger on commit push, so `lint` and `test` still run on every commit. `weekly_rebase` is filtered to only run on tag push, so CircleCI runs the `weekly_rebase` job only when a tag is created.

Example on personal fork:
![image](https://user-images.githubusercontent.com/11583824/85971946-64219c00-b983-11ea-8e9c-e8dc740af218.png)

Do note the additional run on master due to the rebase.